build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -95,4 +100,18 @@
     <!-- SetTelemetryProfile -->
     <SetEnvVar Name="DOTNET_CLI_TELEMETRY_PROFILE" Value="https://github.com/dotnet/cli;$(CommitHash)" />
   </Target>
+
+  <Target DependsOnTargets="Init" Name="RestorePackages">
+
+    <!-- Clean NuGet Temp Cache on Linux (seeing some issues on Linux) -->
+    <!-- TODO: do we still need this?  Seems unnecessary -->
+    <RemoveDir
```

Comment:
Let's kill it and bring back if we see these 'issues'. When it comes back, it does so with an issue URL.